internal/commands/platform/export_internal.go

```diff
@@ -334,46 +325,152 @@ func initPingOneApiClient(ctx context.Context, pingcliVersion string) (err error
 		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: ErrNilContext}
 	}
 
-	pingoneApiClientId, err = profiles.GetOptionValue(options.PingOneAuthenticationWorkerClientIDOption)
-	if err != nil {
-		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: err}
-	}
-	clientSecret, err := profiles.GetOptionValue(options.PingOneAuthenticationWorkerClientSecretOption)
-	if err != nil {
-		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: err}
-	}
-	environmentID, err := profiles.GetOptionValue(options.PingOneAuthenticationWorkerEnvironmentIDOption)
-	if err != nil {
-		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: err}
-	}
+	// Try to get legacy worker credentials first
+	workerClientID, _ := profiles.GetOptionValue(options.PingOneAuthenticationWorkerClientIDOption)
+	workerClientSecret, _ := profiles.GetOptionValue(options.PingOneAuthenticationWorkerClientSecretOption)
+	workerEnvironmentID, _ := profiles.GetOptionValue(options.PingOneAuthenticationWorkerEnvironmentIDOption)
 	regionCode, err := profiles.GetOptionValue(options.PingOneRegionCodeOption)
 	if err != nil {
-		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: err}
+		return err
 	}
 
-	if pingoneApiClientId == "" || clientSecret == "" || environmentID == "" || regionCode == "" {
-		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: ErrPingOneConfigValuesEmpty}
+	if regionCode == "" {
+		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: auth_internal.ErrRegionCodeRequired}
 	}
 
 	userAgent := fmt.Sprintf("pingcli/%s", pingcliVersion)
-
 	if v := strings.TrimSpace(os.Getenv("PINGCLI_PINGONE_APPEND_USER_AGENT")); v != "" {
 		userAgent = fmt.Sprintf("%s %s", userAgent, v)
 	}
 
 	enumRegionCode := management.EnumRegionCode(regionCode)
 
+	if workerClientID != "" && workerClientSecret != "" && workerEnvironmentID != "" {
+		l.Debug().Msgf("Using legacy worker authentication with client credentials")
+
+		pingoneApiClientId = workerClientID
+
+		accessToken, tokenValid := loadLegacyWorkerToken(ctx, workerClientID, workerEnvironmentID)
+
+		if tokenValid && accessToken != "" {
+			l.Debug().Msgf("Using cached access token for legacy worker authentication")
+
+			emptyString := ""
+			apiConfig := &pingoneGoClient.Config{
+				AccessToken:     &accessToken,
+				ClientID:        &emptyString,
```

Comment:
Why set these to empty string instead of just leaving as nil? Same question for below